nucypher/datastore/queries.py

```diff
@@ -53,16 +53,9 @@ def find_expired_policies(ds: Datastore, cutoff: maya.MayaDT) -> DatastoreQueryR
                        writeable=True)
 
 
-def find_expired_treasure_maps(ds: Datastore, cutoff: maya.MayaDT) -> DatastoreQueryResult:
-    return ds.query_by(EncryptedTreasureMap,
-                       filter_field='expiration',
-                       filter_func=lambda expiration: expiration <= cutoff,
-                       writeable=True)
-
-
 @unwrap_records
-def get_work_orders(ds: Datastore) -> List[Workorder]:
-    return ds.query_by(Workorder)
+def get_reencryption_requests(ds: Datastore) -> List[ReencryptionRequest]:
```

Comment:
The prometheus call to this function needs to be updated - https://github.com/fjarri/nucypher/blob/retrieval-protocol/nucypher/utilities/prometheus/collector.py#L131. Only the call itself, **not** the metric key string called "work_orders_gauge".

Reply:
Why not the metric too? Backward compatibility?

Reply:
Yep, backward compatibility for the dashboard.

Reply:
Is "work orders" set in stone there, or will we be able to change it eventually? It creates kind of a confusing situation.

Reply:
Linked to the wrong line (associated but not the exact line), it should be https://github.com/fjarri/nucypher/blob/retrieval-protocol/nucypher/utilities/prometheus/collector.py#L101 i.e. `ursula_work_orders` field.

It's not set in stone. It's more that the Grafana dashboard configuration file would need to be updated as well - https://github.com/p2p-org/nucypher-monitoring/blob/master/grafana/provisioning/dashboards/nucypher.json#L1820. We'll need to coordinate with p2p to update that file - separate from that, we could think about whether we want to distribute that file separate from p2p.

Unsure if @KPrasch has already given this some thought - but I don't think it is necessary to resolve at this moment.

Reply:
Filed #2797 and added a TODO in the code